analytics/tracker_object.rst

```diff
@@ -0,0 +1,543 @@
+    :param string price: **Optional** String with product price.
+    :param string quantity: **Optional** String with product quantity.
+
+.. function:: trackEcommerceOrder(orderId, orderGrandTotal, orderSubTotal, orderTax, orderShipping, orderDiscount);
+
+    Function that tracks Ecommerce order, also tracks all items previously added.
+
+    :param string orderId: **Required** Unique order ID.
+    :param number orderGrandTotal: **Required** Order Revenue grand total  - tax, shipping and discount included.
+    :param number orderSubTotal: **Optional** Order sub total - without shipping.
+    :param number orderTax: **Optional** Order tax amount.
+    :param number orderShipping: **Optional** Order shipping costs.
+    :param number orderDiscount: **Optional** Order discount amount.
+
+.. function:: trackEcommerceCartUpdate(grandTotal);
+
+    Function that tracks shopping cart value. Use this each time there is a change in cart as the last function after
+    adding cart items.
+
+    :param number grandTotal:  **Required** Order Revenue grand total  - tax, shipping and discount included.
+
+.. function:: setEcommerceView(productSKU, productName, categoryName, productPrice);
+
+    Function to track product or category page view, must be followed by ``trackPageView`` function.
+
+    :param string productSKU: **Required** String with product stock-keeping unit.
+    :param string productName: **Optional** String with product name.
+    :param Array<string> productCategory: **Optional** Product category, can be written as Array with up to 5 elements.
+    :param string price: **Optional** String with product price.
+
+
+Custom variables
+----------------
+
+.. function:: setCustomVariable(index, name, value, scope);
+
+    Function that sets a custom variable to be used later.
+
+    :param string index: **Required** Number from 1 to 5 where variable is stored.
+    :param string name: **Required** Name of the variable.
+    :param string value: **Required** Value of the variable.
+    :param string scope: **Required** Scope of the variable, ``"visit"`` or ``"page"``.
+
+.. function:: deleteCustomVariable(index, scope);
+
+    Function that will delete a custom variable.
+
+    :param string index: **Required** Number from 1 to 5 where variable is stored.
+    :param string scope: **Required** Scope of the variable, ``"visit"`` or ``"page"``.
+
+.. function:: getCustomVariable(index, scope);
+
+    Function that will return value of custom variable.
+
+    :param string index: **Required** Number from 1 to 5 where variable is stored.
+    :param string scope: **Required** Scope of the variable, ``"visit"`` or ``"page"``.
+
+.. function:: storeCustomVariablesInCookie();
+
+Function will enable storing ``"visit"`` type custom variables in a first party cookie.
+That will enable getting them via ``getCustomVariable`` function.
+
+
+Custom dimensions
+-----------------
+
+.. function:: setCustomDimension(customDimensionId, customDimensionValue);
```

Comment:
May be worth mentioning, that this function doesn't call backend, just sets CD to be sent when methods like `trackPageView` and similar (conversions, etc...) are used